scripts/who/gho/generate_csv_and_sv.py

```diff
@@ -0,0 +1,308 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script to generate csv and the mcf for the StatisticalVariables. """
+
+import requests
+import json
+import csv
+import re
+import os
+from absl import flags
+from absl import app
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    'raw_data_dir', '',
+    'Path to folder that contains the raw json data from the indicators.')
+flags.DEFINE_string(
+    'schema_mapping', '',
+    'path to a json file containing mapping of dimensions and indicators to their corresponding schema dcid.'
+)
+flags.DEFINE_string('output_dir', '',
+                    'directory to output the csv and stat var mcf to.')
+VALUES_TO_DROP = {
+    "No data", "Data not available", "Not available", "Not applicable",
+    "*Filler*"
+}
+
+
+def get_time_period(entry):
+    """For a data entry, get the time period.
+    Args:
+        entry: an entry from the WHO indicator api eg. {
+          "Id": 13233998,
+          "IndicatorCode": "EMFLIMITELECTRIC",
+          "SpatialDimType": "COUNTRY",
+          "SpatialDim": "AUS",
+          "TimeDimType": "YEAR",
+          "TimeDim": 2014,
+          "Dim1Type": "EMFEXPOSED",
+          "Dim1": "EMFPUBLIC",
+          "Dim2Type": "EMFFREQUENCY",
+          "Dim2": "EMFLOW",
+          "Dim3Type": null,
+          "Dim3": null,
+          "DataSourceDimType": null,
+          "DataSourceDim": null,
+          "Value": "[5]/[10]",
+          "NumericValue": null,
+          "Low": null,
+          "High": null,
+          "Comments": "5 kV/m : continuous exposure 5 - 10 kV/m : exposure limited to a few hours a day > 10 kV/m : exposure limited to a few minutes a day, provided the induced current density does not exceed 2 mA/m2 and precautions are taken to prevent hazardous indirect coupling",
+          "Date": "2016-05-27T13:50:19.3+02:00",
+          "TimeDimensionValue": "2014",
+          "TimeDimensionBegin": "2014-01-01T00:00:00+01:00",
+          "TimeDimensionEnd": "2014-12-31T00:00:00+01:00"
+        }
+    Returns:
+        time_period as a string
+    """
+    time_begin = entry.get("TimeDimensionBegin", "").split("-")
+    time_begin_year = time_begin[0]
+    time_end = entry.get("TimeDimensionEnd", "").split("-")
+    time_end_year = time_end[0]
+    time_period = "P1Y"
```

Comment:
Wonder if using standard Date/Time parsers would be more reliable.  Have a TODO maybe.

Reply:
added a TODO